src/ui/buttons/props.js

```diff
@@ -696,6 +719,56 @@ export function normalizeButtonStyle(
   };
 }
 
+export function validateButtonMessage(
```

Comment:
Consider `normalizeButtonMessage` to follow the naming convention of some of the other functions in the file.

Reply:
can do